tests/integration/security/testdata/beta-mtls-permissive.yaml

```diff
@@ -1,15 +1,3 @@
-# Global PeerAuthentication can be removed for this test, once we remove the (alpha) mesh policy
```

Comment:
Can you add a new test case instead of changing the existing one? Should be very simple to just duplicate the test you're changing here.

Reply:
Thanks, updated!